src/core/calc.c

```diff
@@ -7,12 +7,11 @@ int calc_adjust_with_percentage(int value, int percentage)
 
 int calc_percentage(int value, int total)
 {
+    int ret = 0;
```

Comment:
Why introduce a new variable when you can use multiple return statements? Ok, it saves an `else` block, but I find that more readable than setting a variable to return later.